lib/src/component_declaration/flux_component.dart

```diff
@@ -34,7 +34,7 @@ part 'flux_component.over_react.g.dart';
 /// __Example:__
 ///
 /// ```dart
```

Comment:
While we're in here, can we update the example to show the new boilerplate as well, which is what we want to recommend going forward?
```suggestion
/// __Example:__
///
/// ```dart
/// class YourComponentProps = UiProps 
///     with FluxUiPropsMixin<YourFluxActionsClass, YourFluxStoreClass>;
/// ```
///
/// __Example (legacy boilerplate):__
///
/// ```dart
```

Reply:
For sure, good thought!